baselines/doc/source/conf.py

```diff
@@ -37,7 +37,7 @@
 author = "The Flower Authors"
 
 # The full version, including alpha/beta/rc tags
-release = "1.9.0"
+release = "1.11.0"
```

Comment:
```suggestion
release = "1.10.0"
```

Reply:
@danieljanes But that would make the baselines docs inconsistent with the rest, is that what we want?